x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -104,16 +108,37 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
       : AUTOMATIC_UPDATES_OFF_CHECKBOX_LABEL;
 
     const saveButtonEnabled =
-      (fetchedNote ? note !== fetchedNote.note : note !== '') ||
-      manifestVersion !== deployedVersion;
+      (fetchedNote ? note !== fetchedNote.note : note !== '') || // Note has changed
+      manifestVersion !== deployedVersion || // User selected a different version
+      isDeployedVersionDifferentFromSelected; // User entered the screen with deployed version that is different from default, selected one
```

Comment:
I think this is redundant with the line above.  `manifestVersion` is actually the `selectedManifestVersion` (i'd suggest to rename it to help understanding), and if it is initialised correctly, there is no need for `isDeployedVersionDifferentFromSelected`

something like this:
```ts
    const [selectedManifestVersion, setSelectedManifestVersion] = useState(
      deployedVersion === 'latest' ? 'latest' : selectedDate.format(internalDateFormat)
    );
```

this fixes the issue when the user opens the page with disabled auto update, but `selectedManifestVersion` is out of sync with `selectedDate` (which synchronisation happens when the user disables the auto update in `toggleAutomaticUpdates()`)

Reply:
Thanks a lot for this comment. I felt like I was overcomplicating things but didn't see your solution to the problem. Much appreciated ;)

I've renamed the variable, used your solution but kept logic that dictates when to display the unsaved changes modal - a8d8cbb40668073a1586397437a63bb578fb9195

Reply:
awesome, thank for the changes! 🚀